config/locales/en.yml

```diff
@@ -779,10 +782,13 @@ en:
         ad_zones: Control the types of ads that should be stitched into episodes in this feed.
     form_apple_config:
       title: Apple Subscriptions
-      description: In order to publish episodes through Apple's Podcast Connect API, ensure you've delegated access to the PRX account in Podcast Connect.
```

Comment:
this isn't true - you need an API key from whatever apple account, doesn't have to be the PRX account